cmd/dcmd/commands.go

```diff
@@ -35,3 +105,25 @@ var contFlag = cli.StringFlag{
 	Name:  "cont",
 	Usage: "UUID for the new container.",
 }
+
+var objLoFlag = cli.Uint64Flag{
```

Comment:
For a developer interface this is ok, for users we should be generating OIDs internally and mapping them to names so the user never sees the OID (unless they want to look it up). This is the direction I was originally planning to go with this object interface, but now I see we need both, a bare object interface and an different interfaces for higher level things like files or Put/Get style KV values.